### PR TITLE
Fix blocking error in ssh key

### PR DESCRIPTION
**Pull request by MohamedElmdary**
## Issues
- https://github.com/threefoldtech/grid_weblets/issues/1199

### CODE REVIEW

**Review by AhmedHanafy725 — COMMENTED** [review]
can we also check the balance before setting sshkey. And if the user doesn't have enough tokens, show him the QRcode and ask him to fund his wallet.

**Comment by MohamedElmdary** [comment]
@AhmedHanafy725 
Update the behaviour of qrcode to be displayed whenever seeds are valid regarless sshkey. So the user now is allowed to fund his wallet just fine.

File: src/App.svelte [hunk 1]
```diff
@@ -9,7 +9,6 @@
   import Peertube from "./elements/peertube/Peertube.wc.svelte";
   import Funkwhale from "./elements/funkwhale/Funkwhale.wc.svelte";
   import Mattermost from "./elements/Mattermost/Mattermost.wc.svelte";
-  // import Mastodon from "./elements/Mastodon/Mastodon.wc.svelte";
```
Comment (AhmedHanafy725) [inline src/App.svelte]: the changes here in this file are not related to this pr, please get it back

Comment (MohamedElmdary) [reply src/App.svelte]: alright @AhmedHanafy725

**Review by AhmedHanafy725 — APPROVED** [review]
